src/css/responsive/desktop.styl

```diff
@@ -8,3 +8,68 @@ $header-height = 84px
 
 #main-content
   margin-top: $header-height
+
+.dropdown-icon-holder
+  text-align: center
+
+// PRODUCT
+
+#product
+  #product-row
+    min-height: calc('100vh - ' + $header-height)
+
+#product-details-column
+  border-right: $border-style
+  border-top: 0
+  order: -1
+  align-self: initial
+
+#product-details-header
+  cursor: default
+
+#product-details-holder
+  flex-basis: auto
+  flex-grow: 1
+
+#product-details
+  height: auto
+  flex-basis: auto
+  flex-grow: 1
+
+.hide-details
+  display: none
+
+.show-details
+  display: none
+
+#product
+  &.details-active
+    #product-details
+      height: auto
+      flex-basis: auto
+      flex-grow: 1
+    .hide-details
+      display: none
+    .show-details
+      display: none
+
+.product-gallery-item
+  padding: 0 $margin-small
+  img
+    max-width: 30vw
+
+.gallery-prev
+  left: $margin-basic
+
+.gallery-next
+  right: $margin-basic
+
+$dot-diameter = 18px
```

Comment:
I feel like variables should be hoisted to top of files so easy to play with, or even put in the the main.styl and then called inside responsive styl files

Reply:
yea you can move that up if you like, it should be at the top for sure, with probably a more semantic name. i usually put sizing vars inside responsive in case they change at breakpoints.